opentelemetry-sdk/src/logs/logger_provider.rs

```diff
@@ -53,13 +69,18 @@ impl opentelemetry::logs::LoggerProvider for SdkLoggerProvider {
     }
 
     fn logger_with_scope(&self, scope: InstrumentationScope) -> Self::Logger {
-        // If the provider is shutdown, new logger will refer a no-op logger provider.
+        // If the provider is shutdown, new logger will refer a shutdown no-op logger provider.
         if self.inner.is_shutdown.load(Ordering::Relaxed) {
             otel_debug!(
-                name: "LoggerProvider.NoOpLoggerReturned",
+                name: "LoggerProvider.ShutdownNoOpLoggerReturned",
```

Comment:
I have renamed old `NoOpLoggerReturned` to `ShutdownNoOpLoggerReturned` should i change it back to original name. Renamed so we can distinguish between shutdown and disabled case

Reply:
I have used original name and added reason attribute